expo-maps/ios/ExpoMaps/ExpoAppleMapsModule.swift

```diff
@@ -12,6 +12,15 @@ public class ExpoAppleMapsModule: Module {
       prop("mapType") { (view: AppleMapsView, mapType: MapType) in
         view.setMapType(mapType: mapType)
       }
+      prop("showCmpass") { (view: AppleMapsView, enable: Bool) in
+        view.setShowCompassButton(enable: enable)
```

Comment:
Same suggestion as for Android
```suggestion
        view.setShowCompass(enable: enable)
```